tester_app/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,30 @@
+# Shown below are the defaults for configuration
+ReactOnRails.configure do |config|
+  # Client bundles are configured in application.js
+
+  # Server rendering:
+  # Server bundle is a single file for all server rendering of components.
+  # It is important to set this to "" if you are not doing server rendering to avoid an extraneous log warning
+  # that the default file of server-bundle.js does not exist.
+  config.server_bundle_js_file = "app/assets/javascripts/generated/server-bundle.js" # This is the default
```

Comment:
this should be "" for non-server rendering, right? otherwise you get a message that the file is missing possibly in the logs.